### PR TITLE
Updated AD UserGroup and OpenScap Tests appropriately.

### PR DESCRIPTION
**Pull request by kbidarkar**
*) OpenScap: Added deletion testcases
*) OpenScap: Removed irrlevant testcases.
*) ADUserGroup: Added deletion testcase.
*) Updated stubs and added @steps for various testcases.

### CODE REVIEW

File: tests/foreman/ui/test_openscap.py [hunk 1]
```diff
@@ -113,37 +258,14 @@ def test_search_policies(self):
...
-
-        @Assert: Whether performing Ad-Hoc Audit of a given
-        Host/System is possible.
-
-        @status: Manual
-
-        """
-
-    @stubbed()
-    def test_audit_targeted_hosts(self):
```
Comment (kbidarkar) [inline tests/foreman/ui/test_openscap.py]: This is an invalid testcase.

File: tests/foreman/ui/test_openscap.py [hunk 2]
```diff
@@ -113,37 +258,14 @@ def test_search_policies(self):
...
 
-        @Assert: Whether searching OpenScap policies is possible.
-
-        @status: Manual
+        @Steps:
 
-        """
-
-    @stubbed()
-    def test_audit_adhoc_hosts(self):
```
Comment (kbidarkar) [inline tests/foreman/ui/test_openscap.py]: Removed as Ad-hoc audit is a future feature as per the github link https://github.com/OpenSCAP/foreman_openscap [future features]

File: tests/foreman/ui/test_adusergroups.py [hunk 3]
```diff
@@ -17,7 +17,26 @@ def test_create_ldap_auth_withAD(self):
         1. Create a new LDAP Auth source with AD
         2. Fill in all the fields appropriately for AD
 
-        @Assert: Whether creating LDAP Auth with AD is successful
+        @Assert: Whether creating LDAP Auth with AD is successful.
+
+        @status: Manual
```
Comment (elyezer) [inline tests/foreman/ui/test_adusergroups.py]: Saw that you have changed from `status` to `Status` in this PR, maybe you may want to do the same here.

**Comment by elyezer** [comment]
ACK, pending comment (not a blocker).

File: tests/foreman/ui/test_adusergroups.py (hunk 3 above)
Comment (kbidarkar) [inline tests/foreman/ui/test_adusergroups.py]: oh! might have missed. :) fixing.

File: tests/foreman/ui/test_openscap.py [hunk 4]
```diff
@@ -101,9 +240,15 @@ def test_search_content(self):
 
         @Feature: OpenScap - Search.
 
+        @steps:
+
+        1. Create an openscap content.
+        2. Provide valid data-stream.xml file available from
+        scap-security-guide. (ssg-rhel6-ds.xml)
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_openscap.py]: Nitpick: indent this.

    1. Create an openscap content.
    2. Provide valid data-stream.xml file available from
       scap-security-guide. (ssg-rhel6-ds.xml)

**Comment by Ichimonji10** [comment]
ACK pending comments (none are blockers)

File: tests/foreman/ui/test_openscap.py (hunk 4 above)
Comment (kbidarkar) [inline tests/foreman/ui/test_openscap.py]: oops! totally missed it. :)

Comment (Ichimonji10) [inline tests/foreman/ui/test_openscap.py]: bump

**Comment by Ichimonji10** [comment]
Rebase, please? Ping me when I hop online tomorrow morning, @kbidarkar, and I'll merge this.

**Comment by Ichimonji10** [comment]
Thank you.

**Pull request merged**